packages/orchestration/src/examples/auto-stake-it.contract.js

```diff
@@ -33,16 +72,42 @@ const contract = async (
   zone,
   { chainHub, orchestrateAll, vowTools },
 ) => {
-  const makeStakingTap = prepareStakingTap(
-    zone.subZone('stakingTap'),
-    vowTools,
-  );
   const makePortfolioHolder = preparePortfolioHolder(
     zone.subZone('portfolio'),
     vowTools,
   );
 
-  const { makeAccounts } = orchestrateAll(flows, {
+  /**
+   * Provides a {@link TargetApp} that reacts to an incoming IBC transfer.
+   */
+  const makeStakingTap = zone.exoClass(
+    'StakingTap',
+    M.interface('AutoStakeItTap', {
+      receiveUpcall: M.call(M.record()).returns(M.undefined()),
+    }),
+    /** @param {StakingTapState} initialState */
+    initialState => {
+      mustMatch(initialState, StakingTapStateShape); // TODO use opts.stateShape
+      return harden(initialState);
+    },
+    {
+      /**
+       * Transfers from localAccount to stakingAccount, then delegates from the
+       * stakingAccount to `validator` if the expected token (remoteDenom) is
+       * received.
+       *
+       * @param {VTransferIBCEvent} event
+       */
+      receiveUpcall(event) {
+        trace('receiveUpcall', event);
+        const { localAccount, stakingAccount, config } = this.state;
+        // eslint-disable-next-line no-use-before-define -- defined by orchestrateAll, necessarily after this
+        orchFns.autoStake(localAccount, stakingAccount, config, event);
```

Comment:
I wanted to pass `this.state` here, and call it ~`continuingOfferCtx` on the other end, but was greeted with `Error: cannot serialize Remotables with accessors like "localAccount"`

Reply:
you might try passing a copy with normal properties rather than accessors `{... this.state }`.